api/models/Centres.js

```diff
@@ -5,6 +5,14 @@ var ValidationError = require('../lib/exceptions/ValidationError');
 var BedCountService = require('../services/BedCountService');
 var LinkingModels = require('sails-linking-models');
 
+// helpers
+const unreconciledMovementReducer = (movements, gender, direction) => movements.reduce((reduced, m) => {
+  if (m.gender === gender && m.direction === direction) {
+    reduced.push({ id: m.id, cid_id: m.cid_id });
+  }
```

Comment:
Please may you explain the reasoning behind sending through both the id and the cid_id. I would suggest that only the cid_id is required.